source/winVersion.py

```diff
@@ -211,10 +213,14 @@ def isUwpOcrAvailable():
 	return os.path.isdir(UWP_OCR_DATA_PATH)
 
 
-def isFullScreenMagnificationAvailable() -> bool:
-	"""
-	Technically this is always False. The Magnification API has been marked by MS as unsupported for
-	WOW64 applications such as NVDA. For our usages, support has been added since Windows 8, relying on our
-	testing our specific usage of the API with each Windows version since Windows 8
-	"""
-	return getWinVer() >= WIN8
+if NVDAState._allowDeprecatedAPI():

```

Comment:
can this please use the standard `__getattr__` methoud outlined in [deprecations.md](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/deprecations.md#deprecations-1)?

Reply:
I did it like this initially as the deprecated symbol is a function, not a constant or a variable, using gui.quit as precedence. If an add-on or other part of NVDA calls winVersion.isFullScreenMagnificationAvailable using the proposed deprecation method, Python will raise attribute error, which amounts to removing the function altogether from the module.

Reply:
Makes sense

Reply:
I think the `gui.quit` example did come before this was standardised though, so I wouldn't use it as precedent

Reply:
I think this should give us a motivation to refine the getattr method when dealing with deprecated functions and class methods. I'll follow the suggested process, and then will create a new GitHub issue.